### PR TITLE
Script Network RPC Setup

### PR DESCRIPTION
**Pull request by dheeraj2414**
Script Network RPC Setup

### CODE REVIEW

**Review by ligi — CHANGES_REQUESTED** [review]
not compatible with the EIP

File: _data/chains/eip155-742.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "decimals": 18
+  },
+  "infoURL": "https://token.script.tv",
+  "shortName": "SPAY",
+  "chainId": 742,
+  "networkId": 742,
+  "explorers": [{
+    "name": "Script Explorer",
+    "url": "https://explorer.script.tv",
+    "standard": "EIP3091"
```
[the remark this answers is not in the document]
Comment (dheeraj2414) [reply _data/chains/eip155-742.json]: Recommended changes has been done standard replace with none

[the remark this answers is not in the document]
Comment (dheeraj2414) [reply _data/chains/eip155-742.json]: Hi @ligi , may i know what the reason behind fail this action (https://github.com/ethereum-lists/chains/actions/runs/5595477102/jobs/10239894257). I have added json object according to example. Please let me know. Thanks 
Screenshot from 2023-07-19 19-09-56

**Comment by surgarshorts** [comment]
Sorry reviewing hyperlinks to learn and understand

**Comment by ligi** [comment]
diff is empty - closing

**Pull request closed without merging**